example/relational/README.md

```diff
@@ -162,13 +162,15 @@ SortKey is
   ⟿ keyword:theory
 */
 type Keyword struct {
-	HashKey dynamo.IRI `dynamodbav:"prefix,omitempty"`
+	HashKey dynamo.IRI `dynamodbav:"prefix,omitempty"` => PartitionKey?
 	SortKey dynamo.IRI `dynamodbav:"suffix,omitempty"`
 }
 ```
 
 There are only a few limited ways to query data efficiently from DynamoDB. The composite sort key together with `dynamo.IRI` (`curie.IRI`) data type let application retrieve hierarchy of related items using range queries with expressions `begins_with`, `between`, `>`, `<`, etc. The `dynamo` library automates construction of these expressions.
 
+=> what would be the impact on write in this case?
+==> I guess an article could have multiple keywords and so when inserting / updating the article, those keywords would need to be refreshed?
```

Comment:
Yes, exactly! The Adj List Pattern requires from an application to explicitly manage relations by adding/removing them. For example like here https://github.com/fogfish/dynamo/blob/7c624aed19be027d2e20cb182b2fdd1d1cbf3687/example/relational/main.go#L278

which is very similar in RBDMS world when your maintain N-to-M or 1-N relations.